ocaml/xapi/xapi_gpumon.ml

```diff
@@ -79,21 +79,21 @@ module Nvidia = struct
       let domid = Db.VM.get_domid ~__context ~self:vm |> Int64.to_int in
       Db.VGPU.get_resident_on ~__context ~self:vgpu
       |> fun self -> Db.PGPU.get_PCI ~__context ~self
-      |> fun self -> Db.PCI.get_pci_id ~__context ~self
-      |> Gpumon_client.Client.Nvidia.get_vgpu_metadata dbg domid
-      |> function
-        | []      -> []
-        | [meta]  -> [key, Stdext.Base64.encode meta]
-        | _::_    -> failwith @@ Printf.sprintf
-          "%s: VM %s (dom %d) has more than one NVIDIA vGPU (%s)"
-          this (Ref.string_of vm) domid __LOC__
+                     |> fun self -> Db.PCI.get_pci_id ~__context ~self
```

Comment:
Indentation here has gone wild as well... it would be nice to see if there is a flag to stop this behaviour in `ocp-indent`

Reply:
I think it assumes it's part of the same function, which would make sense. Not sure how you'd differentiate between that without parentheses/begin-end

Reply:
By the use of the `|>`

Reply:
But what if the code above was like this:
`|> (function
  | x -> do_this
  | y -> some_function
        |> followed_by_this_one)`
I'm not sure how it would parse that without the parentheses

Reply:
I would expect to use parentheses in that case in fact

Reply:
I think that's currently the situation; the original function wasn't wrapped in parentheses, so ocp-indent assumed the next `|> function` was part of the same lambda function, hence the weird indent.

Reply:
Yes, I'd like to understand if it is right or not :)